src/Illuminate/Mail/Mailer.php

```diff
@@ -210,7 +210,7 @@ public function render($view, array $data = [])
      * @param  string|array|MailableContract  $view
      * @param  array  $data
      * @param  \Closure|string  $callback
-     * @return void
+     * @return mixed
```

Comment:
If all the callers of send() was changed to not use the return value of send(), why should this method start returning a value?

Reply:
that's a very good question and it shows a bit of inconsistency introduced along with the `Mailables` more than a year ago.

Following `sendMailable` method leads to a path where a value is actually returned (non-null), so there's either a need to adjust this method or its signature.

Reply:
In the earlier version, it is possible to get the number of recipient that you can send the email to if you're not using any queue. But to avoid not making any breaking while telling everyone that you shouldn't really rely on the return output it was made to `@return void`.

Reply:
@crynobone thanks, that makes sense. 
So what you're saying is, that we basically should remove the `return` from this method completely and restore, proper at this point, `@returns void` hint in the next release. 
Do I get it right?

Reply:
Laravel use return void sometimes to indicate it may or can return something, but you shouldn't rely on the return value unless you really sure what you're doing

Reply:
which is a very old convention and we can improve it to provide more value to the dev consuming the code, would you agree?

Reply:
Not really, since Laravel isn't keen on going strict typehint

Reply:
In this instance you'd be providing "more value to the dev" by removing a feature that is very useful for debugging.

Reply:
@36864 I'm not sure I follow. Can you share an example of what you mean?